code/datums/outfits/outfit_admin.dm

```diff
@@ -220,24 +220,40 @@
 
 /datum/outfit/admin/nt_undercover
 	name = "NT Undercover Operative"
+	var/access_level = "Emergency Response Team Leader"
 	// Disguised NT special forces, sent to quietly eliminate or keep tabs on people in high positions (e.g: captain)
+	//nerfed ghost version, doesn't get the op centcom combat implant, less access
 
 	uniform = /obj/item/clothing/under/color/black
 	back = /obj/item/storage/backpack
 	belt = /obj/item/storage/belt/utility/full/multitool
-	gloves = /obj/item/clothing/gloves/combat
+	gloves = /obj/item/clothing/gloves/color/yellow
```

Comment:
Is this because combat gloves do things that insulated gloves don't?
Or is this just to help them blend in more?

Reply:
Blend in more. Combat gloves also offer heat protection, but that is so rarely used it's not a factor.

Reply:
It's for blending in more. Combat gloves are fairly rare and the fire resistance rarely comes up, imo. Nobody bats an eye at am assistant with insulated gloves.